reconcile/quay_mirror.py

```diff
@@ -116,9 +122,10 @@ def process_sync_tasks(self):
 
         summary = self.process_repos_query()
         sync_tasks = defaultdict(list)
-        for org, data in summary.items():
+        for org_key, data in summary.items():
+            org = org_key[1]
```

Comment:
Should we use a NamedTuple for the key? `org_key.org` is better than `org_key[1]`.

Reply:
fixed